packages/transaction-controller/src/TransactionController.ts

```diff
@@ -387,6 +388,30 @@ export class TransactionController extends BaseController<
     });
   }
 
+  /**
+   * Creates approvals for all unapproved transactions persisted.
+   */
+  initApprovals() {
+    const {
+      networkId,
+      providerConfig: { chainId },
+    } = this.getNetworkState();
```

Comment:
As we do this in a few places, is it worth creating a `private getChainAndNetworkId` that returns an object with both values in?

Reply:
Done.